.ruby-version

```diff
@@ -1 +1 @@
-2.5.5
+2.6.4
```

Comment:
Do we need to use a newer Ruby version for this repo?

Reply:
No we don't. I updated this to keep it in line with the rest of our projects (b79d8d2587810cf59124da7d65eb6863d124cdf8).

Do you think it's a problem? An out-of-scope change on a PR that should be as little as possible given it targets the release branch?

Reply:
I personally would keep that change, as I agree it's nice to use the same ruby version across all our tooling.
I did a similar update in the past on another PR and repo (can't remember which one, but we discussed it in PR comments with Gio at that time).

The benefits of using the same ruby across our repos whenever possible is to avoid having to `rbenv install` and `bundle install` all the gems again (duplicating all the downloads and size on disk for the gems for usually no main benefit), while I don't really see any major drawback.

Reply:
kk works for me if we're making things _more_ consistent!